x-pack/plugins/security_solution/public/common/lib/telemetry/constants.ts

```diff
@@ -66,6 +66,8 @@ export enum TelemetryEventTypes {
   EntityDetailsClicked = 'Entity Details Clicked',
   EntityAlertsClicked = 'Entity Alerts Clicked',
   EntityRiskFiltered = 'Entity Risk Filtered',
+  EntityStoreEnablement = 'Entity Store Enablement',
+  EntityStoreInit = 'Entity Store Initialized',
```

Comment:
I was expecting these event names to be more UI specific. Like `Enable button toggle` or `Enable button clicked`

Reply:
done